file-proxy/encrypted-file-proxy.js

```diff
@@ -0,0 +1,98 @@
+"use latest"
+
+import Webtask from 'webtask-tools';
+import express from 'express';
+import crypto from 'crypto';
+import multiparty from 'multiparty';
+import formData from 'form-data';
+import request from 'request';
+
+const app = express();
+
+// The upload endpoint
+app.post('/:projectid', (req, res) => {
```

Comment:
I think you mean `projectId`? :)
I also have made some minor adjustments and corrections [here](https://github.com/yusinto/functions/blob/master/file-proxy/encrypted-file-proxy.js)